data/json/harvest.json

```diff
@@ -171,12 +174,16 @@
   {
     "id": "birch_harv",
     "type": "harvest",
-    "entries": [ { "drop": "birchbark", "base_num": [ 2, 8 ] } ]
+    "entries": [ { "drop": "birchbark", "base_num": [ 2, 8 ] }, { "drop": "innerbark_raw", "base_num": [ 1, 4 ] } ]
```

Comment:
It looks like this is related to https://en.wikipedia.org/wiki/Bark_bread and taking the inner piece of the bark may harm the tree, resulting in its death if I am not mistaken. This assumes you are not taking it from twigs or branches.

Reply:
Oh and even if you do use branches/twigs, you have to have a knife to harvest the inner bark. There is no way you can just peel it off.

https://practicalselfreliance.com/birch-bark-flour/

Reply:
For the purpose of sustainability, I'm assuming that 1/6th of all bark 30 cm above the ground and up to however high an average person would be able to reach (about 230 cm?) is removed as a single long but narrow strip.

It's true that you'd need to do a cutting tool to harvest inner bark. Given that currently there's no support for defining required tools/qualities for harvesting certain items via JSON, I'll probably not add bark harvesting until I (or someone else, for that matter) can find time to set up a compiler and figure out the C++ code to add this support.

I'd rather not waste the effort spent on researching the topic of using tree bark as food though, so I've decided to add it as a possible lab sample for plant mutagen making so as to not leave it unobtainable. Let me know if there's any other item group better suitable for putting raw tree bark into, as I couldn't find anything else.